tests/valid/draft-template-old.html

```diff
@@ -910,7 +910,7 @@ <h1 id="rfc.references.2">
 <tr>
 <td class="reference"><b id="I-D.narten-iana-considerations-rfc2434bis">[I-D.narten-iana-considerations-rfc2434bis]</b></td>
 <td class="top">
-<a>Narten, T.</a> and <a>H. Alvestrand</a>, "<a href="https://tools.ietf.org/html/draft-narten-iana-considerations-rfc2434bis-09">Guidelines for Writing an IANA Considerations Section in RFCs</a>", Internet-Draft draft-narten-iana-considerations-rfc2434bis-09, March 2008.</td>
+<a title="Google">Alvestrand, H. T.</a> and <a title="IBM">T. Narten</a>, "<a href="https://tools.ietf.org/html/draft-narten-iana-considerations-rfc2434bis-09">Guidelines for Writing an IANA Considerations Section in RFCs</a>", Internet-Draft draft-narten-iana-considerations-rfc2434bis-09, March 2008.</td>
```

Comment:
These are terrible, and we will have to revert them when the bibxml is repaired, but I don't unless we take the time to pull depending on an external bib service from the tests now, it's unavoidable.

Reply:
The issue is reversed names, or adding the org...?

Reply:
All issues for bibxml not xml2rfc - initials handling, reordering authors (see https://github.com/ietf-tools/datatracker/issues/4973), `<format` tags (later down).

Issues _for_ xml2rfc (will track elsewhere) - not depending on the bibxml service for testing.

Reply:
This has been fixed with #959.
This PR has been rebased and re-generated valid tests with https://github.com/ietf-tools/xml2rfc/pull/895/commits/3eb346823d33dfd80fc7aed1211406513d79d2c1